tofu/imas2tofu/_core.py

```diff
@@ -1487,19 +1487,22 @@ def get_events(self, occ=None, verb=True, returnas=False):
         names, times = names[ind], times[ind]
 
         # print and / or return as list / tuple
-        if verb:
+        if verb is True or returnas is str:
```

Comment:
`if verb or returnas is str` ?

Reply:
returnas is a keyword arg used as a flag to modulate in which form the result should be returned

Reply:
same here : `if verb is True or returnas is str` => `if verb or returnas is str` ?

Reply:
Same answer as before  ;-)